server/middleware.js

```diff
@@ -19,6 +21,36 @@ export function errorHandler(err, req, res, next) {
 }
 
 export function logAll(req, res, next) {
-  console.info('>>> %s %s %s', req.method, req.url, req.path);
+  console.info('>>> %s %s %s %s', new Date().toISOString(), req.method, req.url, req.path);
```

Comment:
request_id ?

Reply:
This one legacy and is not being used. Should I remove it?